pkg/rpc/client/wsclient.go

```diff
@@ -139,14 +139,18 @@ readloop:
 				val = new(result.NotificationEvent)
 			case response.ExecutionEventID:
 				val = new(result.ApplicationLog)
+			case response.MissedEventID:
+				// No value.
 			default:
 				// Bad event received.
 				break readloop
 			}
-			err = json.Unmarshal(slice[0], val)
-			if err != nil || len(slice) != 1 {
-				// Bad event received.
-				break
+			if event != response.MissedEventID {
+				err = json.Unmarshal(slice[0], val)
+				if err != nil || len(slice) != 1 {
```

Comment:
I think `len(slice) != 1` is always true because of check before the `switch`.

Reply:
Yep, looks like.